### PR TITLE
User hist comp

### PR DESCRIPTION
**Pull request by Musique243**

### CODE REVIEW

**Review by colinwilliams91 — APPROVED** [review]
hell yeah big dawg

**Review by Benthusiast — APPROVED** [review]
Hundred percent.

File: client/src/components/userProfiles/UserHistory.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+
+const UserHistory = () => {
+  const [userWins, setUserWins] = useState();
```
Comment (copelka) [inline client/src/components/userProfiles/UserHistory.jsx:6]: initialize state with a value of the same datatype that you expect the variable to hold

File: client/src/components/userProfiles/UserHistory.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  const [userLosses, setUserLosses] = useState();
+  const [userDraws, setUserDraws] = useState();
+
+  useEffect(() => {
+    axios.get('api/user/current')
```
Comment (copelka) [inline client/src/components/userProfiles/UserHistory.jsx:11]: destructure

File: client/src/components/userProfiles/UserHistory.jsx (hunk 1 above)
Comment (colinwilliams91) [reply client/src/components/userProfiles/UserHistory.jsx:6]: oops!

**Pull request merged**